app/Platform/Controllers/PlayerGameController.php

```diff
@@ -85,16 +89,61 @@ public function update(Request $request, User $user, Game $game): void
         $this->authorize('update', [PlayerGame::class, $playerGame]);
     }
 
-    public function destroy(User $user, Game $game): void
+    public function destroy(Request $request, Game $game): JsonResponse
     {
-        $playerGame = $user->playerGames()
-            ->where('game_id', $game->id)
-            ->firstOrFail();
+        /** @var User $user */
+        $user = $request->user();
 
-        $this->authorize('delete', [PlayerGame::class, $playerGame]);
+        (new ResetPlayerProgress())->execute($user, gameID: $game->id);
+
+        return response()->json(['message' => __('legacy.success.reset')]);
+    }
 
-        /*
-         * TODO: detach
-         */
+    public function resettableGames(Request $request): JsonResponse
+    {
+        /** @var User $user */
+        $user = $request->user();
+
+        $resettableGames = $user
+            ->games()
+            ->with('system')
+            ->where('player_games.achievements_unlocked', '>', 0)
+            ->orderBy('Title')
+            ->select(['GameData.ID', 'Title', 'ConsoleID', 'achievements_published', 'player_games.achievements_unlocked'])
+            ->get()
+            ->map(function ($game) {
+                return new PlayerResettableGameData(
+                    id: $game->id,
+                    title: $game->title,
+                    consoleName: $game->system->name,
+                    numAwarded: $game->achievements_unlocked,
+                    numPossible: $game->achievements_published
+                );
+            });
```

Comment:
should this exclude events? `->whereNotIn('system.id', System::getNonGameSystems())`

Reply:
Good catch -- events are now properly excluded in latest.